setup/flex.rst

```diff
@@ -176,6 +176,16 @@ manual steps:
    .. code-block:: terminal
 
        $ composer remove symfony/symfony
+       
+   Now add the ``symfony/symfony`` package to the ``conflict`` section of the project's
+   ``composer.json`` file as `shown in this example of the skeleton-project`_ so that
+   it will not be installed again:
+
+   .. code-block:: json
+
+        "conflict": {
+            "symfony/symfony": "*"
+        },
```

Comment:
I guess this will not be rendered properly as this extra is not a full valid JSON document. What do you think if we used a diff block here instead?